system-tests/README.md

```diff
@@ -0,0 +1,42 @@
+## System tests
+
+System tests run a sample EDC connector with the Identity Hub extension and a DID server with sample DID documents for the connector and an external authority using docker. The test checks the correct functioning of the Identity Hub CLI together with the CredentialsVerifier.
+
+An instance of `CredentialsVerifiar` is injected in the test (hence the `@ExtendWith(EdcExtension.class)` annotation), to verify that another EDC connector is able to retrieve credentials from the Identity Hub after these have beed added using the CLI.
+
+#### Local test resources
+
+Following test resources are provided in order to run system tests:
+
+- A set of Private and Public keys for both the external authority and the EDC connector (identity hub owner) e.g. `system-tests/resources/jwt/authority`.
+
+    ```bash
```

Comment:
please explain what these commands are for (also so that the user understands they don't need to run them just to run the test)